cl/settings/project/corpus_importer.py

```diff
@@ -0,0 +1,14 @@
+import environ
+
+env = environ.FileAwareEnv()
+IQUERY_PROBE_ITERATIONS = env.int("IQUERY_PROBE_ITERATIONS", default=9)
+IQUERY_PROBE_WAIT = env.int("IQUERY_PROBE_WAIT", default=300)
+IQUERY_PROBE_MAX_CONSECUTIVE_FAILURES = env.int(
+    "IQUERY_PROBE_MAX_CONSECUTIVE_FAILURES", default=5
+)
+IQUERY_COURT_BLOCKED_WAIT = env.int("IQUERY_COURT_BLOCKED_WAIT", default=600)
+IQUERY_COURT_TIMEOUT_WAIT = env.int("IQUERY_COURT_TIMEOUT_WAIT", default=10)
```

Comment:
Similar to the previous setting, this one also appears unused. We can likely remove it for consistency.

Reply:
Thanks. yeah, this is unused; it was part of a previous architecture. Now we just abort the task on a timeout and wait for the next iteration.